### DISCUSSION TITLE
Welcome to code-django Discussions!

### DISCUSSION THREAD

**Discussion by almahdi404** [devmahdi404/code-django#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cdevacc1** [comment]
This extension not work if I installed [django](https://github.com/vscode-django/vscode-django) extension too. How can I make it work together?

**Comment by TonL** [comment]
Hello, friend. Good extension, but there are some problems, for example links to defenitions

`"{% url 'somepage' %}"`

if I click here
![image](https://user-images.githubusercontent.com/84812632/202282680-01922461-dd9d-41bd-a180-4cc7763d653a.png)
![image](https://user-images.githubusercontent.com/84812632/202282744-61cf806e-d393-4a9d-9e80-3cb5bbfae7a2.png)
I get an error (File not found)
